pathway-analysis/01-overrepresentation_analysis.Rmd

```diff
@@ -260,11 +319,21 @@ Sometimes folks consider genes from the entire genome to comprise the background
 > 17,980 genes detected in a Microarray study 
 
 Where the key phrase is **genes detected**. 
-If we are unable to detect a gene, we should remove it from our background set.
-In the scRNA-seq module (`01-filtering_scRNA-seq.Rmd` specifically), we filtered out genes that were expressed in less than 200 cells, but all genes that were _detected_ should be in our marker data frame.
 
-```{r detected_genes, live = TRUE}
-detected_genes <- unique(as.character(neuron_markers_df$gene_symbol))
+If we are unable to include a gene because, for example, it had low mean expression in our experiment and therefore was filtered out in our `tidyr::drop_na()` step, we shouldn't include in our background set.  
+
+We can use another function for set operations, `intersect()`, to get our background set.
```

Comment:
I thought this could be a bit more explicit about what `intersect` is doing. It is in the comment, but do people read those?
```suggestion
We can use another function for set operations, `intersect()`, to get our background set of genes that were included in both comparisons.
```